src/Microsoft.ApiDesignGuidelines.Analyzers/Core/CollectionsShouldImplementGenericInterface.cs

```diff
@@ -4,36 +4,76 @@
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Analyzer.Utilities;
+using System.Linq;
```

Comment:
`System` namespaces at the top, and the rest alphabetized (so `Analyzer.Utilities`) would come next.